src/scanner.cc

```diff
@@ -188,18 +188,18 @@ vector<string> names = {
   "empty",
 };
 
-string name(Sym t) { return t < names.size() ? names[t] : "unknown"; }
+static string name(Sym t) { return t < names.size() ? names[t] : "unknown"; }
```

Comment:
Idk if it would help. But try turning `"unknown"` into a `static const std::string`.

Reply:
Removed the check in the end. Totality is the consideration of the indexer now.